marathon-api/marathonapi.py

```diff
@@ -0,0 +1,151 @@
+        """
+        self.logger.info("Reading config file and formatting data.")
+        self.logger.info(config_file)
+        ### open and load json config file
+        try:
+            with open(config_file) as data_file:
+                data = json.load(data_file)
+        except Exception as e:
+            self.logger.error("Error in json file:  %s. Exception is %s" % (config_file, e))
+            sys.exit(1)
+        return data
+
+    def assign_config_data(self, config_file_data, marathon_app_result):
+        """
+        Assign variables from the config file to the marathon app values.
+        We cannot use a dymnamic variable so we are assigning these one at a time.
+        """
+        ### assigning values from json to our marathon app
+        self.logger.info("Assigning json variables to app data")
+        marathon_app_result.cpus                     = config_file_data["cpus"]
+        marathon_app_result.mem                      = config_file_data["mem"]
+        marathon_app_result.instances                = config_file_data["instances"]
+        marathon_app_result.disk                     = config_file_data["disk"]
+        marathon_app_result.cmd                      = config_file_data["cmd"]
+        marathon_app_result.backoff_factor           = config_file_data["backoff_factor"]
+        marathon_app_result.constraints              = config_file_data["constraints"]
+        marathon_app_result.container                = config_file_data["container"]
+        marathon_app_result.dependencies             = config_file_data["dependencies"]
+        marathon_app_result.env                      = config_file_data["env"]
+        marathon_app_result.executor                 = config_file_data["executor"]
+        marathon_app_result.health_checks            = config_file_data["health_checks"]
+        marathon_app_result.labels                   = config_file_data["labels"]
+        marathon_app_result.max_launch_delay_seconds = config_file_data["max_launch_delay_seconds"]
+        #marathon_app_result.ports = config_file_data["ports"]      # You cannot specify both ports and port definitions
+        marathon_app_result.require_ports            = config_file_data["require_ports"]
+        marathon_app_result.store_urls               = config_file_data["store_urls"]
+        marathon_app_result.upgrade_strategy         = config_file_data["upgrade_strategy"]
+        marathon_app_result.uris                     = config_file_data["uris"]
+        marathon_app_result.user                     = config_file_data["user"]                 #updates confirmed
+
+    def list_marathon_apps(self, marathon_server):
+        """
+        This method connects to the marathon server and requests all the
+        Marathon Apps.
+        """
+        self.logger.info("Listing apps running on marathon")
+        current_marathon_apps = marathon_server.list_apps()
+        self.logger.info(current_marathon_apps)
```

Comment:
you shouldn't logger.info here, instead, return the current_marathon_apps (list?) to the cli.py and print there